keanu-project/src/main/java/io/improbable/keanu/util/ProgressBar.java

```diff
@@ -0,0 +1,146 @@
+package io.improbable.keanu.util;
+
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicReference;
+
+import org.apache.commons.io.output.NullOutputStream;
+
+public class ProgressBar {
+
+    public static PrintStream DEFAULT_PRINT_STREAM = System.out;
```

Comment:
is this supposed to be Final?  The name implies so.

Reply:
Ah nice catch. It was final but I changed it and didn't change the case. Changed it.